pkg/compute/init.go

```diff
@@ -355,6 +355,12 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 			Comment: description,
 		})
 		if err != nil {
+			if strings.Contains(err.Error(), "Valid values for 'type' are: 'vcl'") {
```

Comment:
Using `strings.Contains` for checking an error always make me a little nervous due to its brittle nature. Sadly I'm not sure if there's a way around this. I just wanted to vocalise that this might be a problem that resurfaces if the backend API ever changes the text content.

Reply:
Yep, this is not great – we would want an error code to match on in an ideal world. For now this will have to do though, unless anyone has any suggestions.